app/src/main/java/com/paypal/psix/fragments/EventsFragment.java

```diff
@@ -70,4 +61,25 @@ public void onItemClick(AdapterView<?> adapterView, View view, int position, lon
         super.onDestroyView();
         ButterKnife.reset(this);
     }
+
+    private ArrayList<Event> eventsDataSource() {
+        // Demo data.
+        Event[] array = {
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent(),
+                Event.GenerateRandomEvent(), Event.GenerateRandomEvent(), Event.GenerateRandomEvent()
+        };
```

Comment:
Not that it actually matters... but still, this hurts my eyes...

![why not a for loop](https://cloud.githubusercontent.com/assets/821024/6545206/e76ee4fa-c582-11e4-8f15-25953e94ec41.png)